src/check_log.c

```diff
@@ -94,11 +94,6 @@ void srunner_set_tap(SRunner * sr, const char *fname)
     sr->tap_fname = fname;
 }
 
-int srunner_has_tap(SRunner * sr)
```

Comment:
Maybe make this a separate commit, instead of combining multiple unrelated improvements in one commit.